src/commons/modifiers.js

```diff
@@ -18,14 +18,15 @@ export function extractColorValue(props) {
   return Colors[color];
 }
 
-// todo: refactor this and use BACKGROUND_KEY_PATTERN
 export function extractBackgroundColorValue(props) {
   let backgroundColor;
-  _.forEach(Colors, (value, key) => {
-    if (props[`background-${key}`] === true || props[`bg-${key}`] === true) {
-      backgroundColor = value;
-    }
-  });
+
+  const keys = Object.keys(props);
+  const bgProp = keys.find((prop) => Colors.getBackgroundKeysPattern().test(prop));
```

Comment:
I checked in master and I see it did take the last backgroundColor modifier that was passed (even though we don't have a test specifically for it).
I think it's still better to keep parity with the previous behavior, just in case...

Reply:
Done